src/components/UsernameEntry/actions.js

```diff
@@ -60,7 +60,7 @@ export const sendSlackMessage = username => async (dispatch, getState) => {
 	let state = getState();
 	let id = getIDByUsername(username, state.users);
 
-	let storeCode = state.prediction ? state.prediction.id : '';
+	let storeCode = state.actualItem;
```

Comment:
what if actualItem is null? does this fail nicely? Can it be null?

Reply:
It will send a link to the user that fails, all calls to this page are preceded with setting the actual item